src/main/java/org/apache/commons/beanutils2/FluentPropertyBeanIntrospector.java

```diff
@@ -138,9 +138,13 @@ public void introspect(final IntrospectionContext icontext)
                         .getPropertyDescriptor(propertyName);
                 try {
                     if (pd == null) {
-                        icontext.addPropertyDescriptor(createFluentPropertyDescritor(
-                                m, propertyName));
+                        icontext.addPropertyDescriptor(createFluentPropertyDescriptor(
```

Comment:
trivia: in the `private createFluentPropertyDescriptor` second argument was not used